### PR TITLE
One element

### PR DESCRIPTION
**Pull request by courtin**
Implements single-element blade element model for propeller; links to airfoil cd(cl, re) fit.  Prandtl tip loss factor is calculated per qprop theory.  Adds electric motor model and propulsor model, which links the two.

### CODE REVIEW

File: gpkitmodels/GP/aircraft/prop/propeller.py [hunk 1]
```diff
@@ -20,37 +24,142 @@ class Propeller_Performance(Model):
...
+                      ]
+        with SignomialsEnabled():
+            constraints += [TCS([Wa<=V + va]),
+                            #SignomialEquality(Wt,omega*r-vt),
+                            TCS([Wt>=omega*r-vt]),
+                            TCS([T <= rho*B*G*(Wt-eps*Wa)*R]),
+                            #SignomialEquality(vt**2*F**2*(1.+(4.*lam_w*R/(pi*B*r))**2),(B*G/(4.*pi*r))**2),
+                            TCS([vt**2*F**2*(1.+(4.*lam_w*R/(pi*B*r))**2)>=(B*G/(4.*pi*r))**2]),
+
+                            #SignomialEquality(W**2,(Wa**2+Wt**2)),
```
Comment (mjburton11) [inline gpkitmodels/GP/aircraft/prop/propeller.py]: Can we delete all of these?

File: gpkitmodels/GP/aircraft/motor/motor_test.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
+from gpkit import Model, parse_variables, SignomialsEnabled, SignomialEquality, units
```
Comment (mjburton11) [inline gpkitmodels/GP/aircraft/motor/motor_test.py]: I think this file needs to be cleaned up.  I like how you've decided to test the model but I want to just clean up this file.

**Comment by mjburton11** [comment]
@courtin sorry it took me so long to do this,  I just went through and cleaned up some files.  Let me know if you agree with the changes.  I think there's a few things that should be cleaned up before merging.  Looking good though!

File: gpkitmodels/GP/aircraft/motor/motor.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    W                       [lbf]              propulsor weight
+
+
+    """
+    flight_model = Propulsor_Performance
+
+    def setup(self):
+        exec parse_variables(Propulsor.__doc__)
+
+        #self.prop = Actuator_Propeller()
```
[the remark this answers is not in the document]
Comment (bqpd) [reply gpkitmodels/GP/aircraft/motor/motor.py]: (that should be `prop = One_Element_Propeller`)

**Comment by courtin** [comment]
I don't think the results this branch gives are correct so I'm closing this PR

**Pull request closed without merging**